### PR TITLE
[metadata.tvshows.themoviedb.org.python] 1.5.0

### PR DESCRIPTION
**Pull request by pkscout**
### Description

- Nexus only updates to use new VideoInfoTag setters
- general code cleanup
- updated language files

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scrapers/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: metadata.tvshows.themoviedb.org.python/resources/language/resource.language.zh_tw/strings.po [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+"Language-Team: none\n"
+"Language: zh_tw\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+
+msgctxt "Addon Summary"
+msgid "Fetch TV Show metadata from themoviedb.org"
+msgstr ""
```
Comment (razzeee) [inline metadata.tvshows.themoviedb.org.python/resources/language/resource.language.zh_tw/strings.po:21]: This language doesn't seem to actually have translations?

File: metadata.tvshows.themoviedb.org.python/resources/language/resource.language.vi_vn/strings.po [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+"Language-Team: none\n"
+"Language: vi_vn\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+msgctxt "Addon Summary"
+msgid "Fetch TV Show metadata from themoviedb.org"
+msgstr ""
```
Comment (razzeee) [inline metadata.tvshows.themoviedb.org.python/resources/language/resource.language.vi_vn/strings.po:21]: No translations

File: metadata.tvshows.themoviedb.org.python/resources/language/resource.language.uz_uz/strings.po [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+"Language-Team: none\n"
+"Language: uz_uz\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+
+msgctxt "Addon Summary"
+msgid "Fetch TV Show metadata from themoviedb.org"
+msgstr ""
```
Comment (razzeee) [inline metadata.tvshows.themoviedb.org.python/resources/language/resource.language.uz_uz/strings.po:21]: Seems to affect multiple of these @gade01 is that expected?

File: metadata.tvshows.themoviedb.org.python/resources/language/resource.language.zh_tw/strings.po (hunk 1 above)
Comment (gade01) [reply metadata.tvshows.themoviedb.org.python/resources/language/resource.language.zh_tw/strings.po:21]: @razzeee All languages need to be created empty before they can be translated at Weblate.
We have a lot of untranslated languages for metadata.tvshows.themoviedb.org.
It's perfectly safe to add these languages, as all untranslated strings will fallback to en_gb.

File: metadata.tvshows.themoviedb.org.python/resources/language/resource.language.uz_uz/strings.po (hunk 3 above)
Comment (gade01) [reply metadata.tvshows.themoviedb.org.python/resources/language/resource.language.uz_uz/strings.po:21]: @razzeee All languages need to be created empty before they can be translated at Weblate.
We have a lot of untranslated languages for metadata.tvshows.themoviedb.org.
It's perfectly safe to add these languages, as all untranslated strings will fallback to en_gb.

Comment (razzeee) [reply metadata.tvshows.themoviedb.org.python/resources/language/resource.language.uz_uz/strings.po:21]: Oh sorry, probably mixing it up with another translation flow then, we usually have people create new languages in weblate.

Comment (gade01) [reply metadata.tvshows.themoviedb.org.python/resources/language/resource.language.uz_uz/strings.po:21]: Well, it's very true that languages usually are created at Weblate. :)

Weblate use matrix branch for this scraper.
So my guess is that all language folders are copied from matrix to nexus branch in this PR?

**Pull request merged**